### PR TITLE
windows.php.net index is broken

### PR DESCRIPTION
**Pull request by gordol**
`PHP Notice:  Undefined variable: title_page in C:\domains\windows.php.net\templates\header.php on line 4`

Added a default page title var for index, and also prepended an @ to the $title_page variable call in header.php. 

Double fixed.

### CODE REVIEW

**Comment by weltling** [comment]
Thanks for the PR. This mirror repo wasn't synced for a while, but it works now. Could you please check and rebase, if needed?

Thanks.

**Comment by Sobak** [comment]
I can confirm the issue. PR itself looks fine to be merged without rebasing.

File: docroot/index.php [hunk 1]
```diff
@@ -4,6 +4,8 @@
 define('CUR_PATH', dirname(__FILE__) . DIRECTORY_SEPARATOR);
 //define('TPL_PATH', CUR_PATH . 'templates' . DIRECTORY_SEPARATOR);
 
+$title_page = 'Home'
```
Comment (weltling) [inline docroot/index.php]: Semicolon is missing.

**Comment by weltling** [comment]
Changes merged, while patch was not available anymore.

Thanks.

File: docroot/index.php (hunk 1 above)
Comment (gordol) [inline docroot/index.php]: yeah, i caught that way earlier. check latest commit.

september 2014. :)

https://github.com/php/web-windows/pull/3/commits/6463127d10e6357f256f1042325f43e7e0da3107

looks like you figured it out, and merged already though. 👍

Comment (weltling) [inline docroot/index.php]: Yeah, thanks for the PR in any case. We're now also synced with the upstream repo, so PRs can target the current state.

**Pull request closed without merging**